### PR TITLE
Vcr specs for dto refresh

### PR DESCRIPTION
**Pull request by Ladas**
Vcr specs for dto refresh

### CODE REVIEW

File: app/models/manageiq/providers/amazon/network_manager/refresh_parser.rb [hunk 1]
```diff
@@ -323,12 +323,6 @@ def parse_load_balancer_health_check(lb)
     port         = target_match[2].to_i
     url_path     = target_match[3]
 
-    matched_listener = @data.fetch_path(:load_balancer_listeners).detect do |listener|
```
Comment (durandom) [inline app/models/manageiq/providers/amazon/network_manager/refresh_parser.rb]: why is this removed?

File: spec/models/manageiq/providers/amazon/aws_refresher_spec_common.rb [hunk 2]
```diff
@@ -0,0 +1,863 @@
+module AwsRefresherSpecCommon
```
Comment (durandom) [inline spec/models/manageiq/providers/amazon/aws_refresher_spec_common.rb]: 👏 for moving this to a common file

File: app/models/manageiq/providers/amazon/network_manager/refresh_parser.rb (hunk 1 above)
Comment (Ladas) [inline app/models/manageiq/providers/amazon/network_manager/refresh_parser.rb]: need it for passing of the specs, it's removed from the dto refresh already

Comment (durandom) [inline app/models/manageiq/providers/amazon/network_manager/refresh_parser.rb]: hmm, but it removes the listeners?!
so it alters the current refresh code.

Comment (durandom) [inline app/models/manageiq/providers/amazon/network_manager/refresh_parser.rb]: ok, but why don't we need it?
now it looks like we dont save `loadbalancer_listeners` anymore
